### ISSUE TITLE
Additional Images notation

### ISSUE THREAD

**Issue by drbyte** [zcadditions/ZCA-Bootstrap-Template-BETA-#27]
May I suggest rewording "Larger Images" to "Additional Images"? 
https://github.com/zcadditions/ZCA-Bootstrap-Template-BETA-/blob/master/includes/languages/english/bootstrap/product_info.php#L25

As "Larger Images" it tells the visitor that there are multiple larger images available ... but really the visitor doesn't care that they're "larger", but rather that there are "more".

ie:

<img width="178" alt="screen shot 2018-03-24 at 3 25 03 pm" src="https://user-images.githubusercontent.com/404472/37868107-f782a144-2f77-11e8-8544-f0e7c9ee3e4c.png">

**Comment by zcadditions** [comment]
Makes sense to me...
